android-configure

```diff
@@ -38,13 +38,26 @@ case $ARCH in
         ;;
 esac
 
+NDK_PATH=$1
+function make_toolchain {
+    $NDK_PATH/build/tools/make-standalone-toolchain.sh \
+         --toolchain=$TOOLCHAIN_NAME-$CC_VER \
+        --arch=$ARCH \
+        --install-dir=$TOOLCHAIN \
+        --platform=android-21
+}
+
 export TOOLCHAIN=$PWD/android-toolchain
-mkdir -p $TOOLCHAIN
-$1/build/tools/make-standalone-toolchain.sh \
-    --toolchain=$TOOLCHAIN_NAME-$CC_VER \
-    --arch=$ARCH \
-    --install-dir=$TOOLCHAIN \
-    --platform=android-21
+if [ -d "$TOOLCHAIN" ]; then
+    read -r -p "Android toolchain already exists. Do you want to replace it?  [y/N] " response
+    case "$response" in 
+        [Yy]) 
+            rm -rf $TOOLCHAIN
```

Comment:
The `rm -rf` makes me nervous.  Can you at least wrap it in double quotes so spaces in the path don't cause bad behavior?